schedule_scaling/main.py

```diff
@@ -66,14 +67,20 @@ def parse_schedules(schedules, identifier):
         return []
 
 
-def get_delta_sec(schedule):
+def get_delta_sec(schedule, timezone_name=None):
     """ Returns the number of seconds passed since last occurence of the given cron expression """
+    # localize the time to the provided timezone, if specified
+    if not timezone_name:
+        tz = None
+    else:
+        tz = dateutil.tz.gettz(timezone_name)
+
     # get current time
-    now = datetime.now()
+    now = datetime.now(tz)
     # get the last previous occurrence of the cron expression
     time = croniter(schedule, now).get_prev()
     # convert now to unix timestamp
-    now = now.replace(tzinfo=timezone.utc).timestamp()
+    now = now.timestamp()
```

Comment:
If no timezone is specified (so things are like they were before) do you still need the `replace(tzinfo=timezone.utc)` step?